kernel/kernel-api/src/main/java/io/delta/kernel/internal/ScanBuilderImpl.java

```
@@ -85,4 +88,10 @@ public Scan build() {
         dataPath,
         snapshotReport);
   }
+
+  @Override
+  public PaginatedScan buildPaginated(
+      long pageSize, Row pageToken, ColumnarBatch logReplayHashsets) {
+    return null;
```

for now, let's just throw `UnsupportedOperationException("not implemented")`